package.json

```
@@ -2,17 +2,16 @@
   "name": "apprentice-at",
   "version": "0.0.1",
   "description": "An online listing of apprenticeship opportunities.",
-  "engines": {
-    "node": "12.0.0"
-  },
   "scripts": {
-    "build": "npm run sass:build && npm run copy:public && npm run concat:js && grunt build && eleventy",
+    "autoprefix": "postcss dist/assets/css/style.css -u autoprefixer -d dist/assets/css",
+    "build": "npm run sass:build && npm run copy:public && npm run concat:js && eleventy && npm run autoprefix",
```

```suggestion
    "build": "run-s clean:all sass:build copy:public concat:js eleventy autoprefix",
```
It looks like `clean:all` was missing from the build step. This suggestion adds it, and since the command was growing kind of long, uses `run-s` to run each command serially (as opposed to `run-p` to run them in parallel). 

The only caveat to this suggestion is that we'll need to add a specific `eleventy` npm script too:

```
"eleventy": "eleventy",
```